requirements/base.in

```diff
@@ -1,6 +1,8 @@
 # Core requirements for using this application
 
-celery >= 3.1,<5.0
+celery >= 3.1,<4.0
 Django                    # Web application framework
+django-celery==3.2.1
```

Comment:
Djano-celery is needed for ChordableDjangoBackend, [and its incompatible with celery 4](https://github.com/celery/django-celery/blob/master/requirements/default.txt)

Upon upgrading to celery 4, we''ll need to switch the ChordableDjangoBackend's base class from [djcelery.backends.database:DatabaseBackend](https://github.com/celery/django-celery/blob/master/djcelery/backends/database.py#L13) to [django-celery-results.backends.database:DatabaseBackend](https://github.com/celery/django-celery-results/blob/master/django_celery_results/backends/database.py#10)